onebusaway-android/src/main/java/org/onebusaway/android/ui/ReportProblemFragmentBase.java

```diff
@@ -55,6 +55,13 @@
      */
     GoogleApiClient mGoogleApiClient;
 
+    /**
```

Comment:
Yeah, I think I'd avoid using the abstract method just for the case of it being used in the future, and just focus on the current use.  If you implement this method as a normal base class method, future subclasses can simply override it to implement different validation logic, if needed.  Or, just leave it in the subclasses.